todo.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const addContent = document.getElementById('output');
+const addText = document.getElementById('text_input');
+const addButton =document.getElementById('trigger');
+const tasks = [];
+//追加ボタンをクリックした時の挙動 テキストデータもリセット
+addButton.addEventListener('click', () => {
+  const addTextValue = addText.value;
+  const task = {
+    text: addTextValue,
+    status: '作業中'
+    };
+  tasks.push(task);
+  addText.value = '';
+  taskProcessing();
+  });
+//tasksの中身をhtml上に表示
+const taskProcessing = () => {
+  while(addContent.firstChild){
+    addContent.removeChild(addContent.firstChild);
+  }
+
+  tasks.forEach((todo,number) => {
```

Comment:
`number` としてしまうと抽象的過ぎるので、`index` あたりでよさそうに思いました。
```suggestion
  tasks.forEach((todo, index) => {
```